src/ClientManager/Travely.ClientManager.Repository/Entity/Turist.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text;
+
+namespace Travely.ClientManager.Repository.Entity
+{
+    [Table("Turist")]
+    public class Turist : BaseEntity
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string PhoneNumber { get; set; }
+        public string Email { get; set; }
```

Comment:
Mark all option fields as nullable (for example `public string? Email { get; set; }`)